packages/js/src/dynamic-blocks/breadcrumbs/block.json

```diff
@@ -0,0 +1,21 @@
+{
+  "$schema": "https://schemas.wp.org/trunk/block.json",
+  "apiVersion": 3,
+  "version": "22.8",
+  "name": "yoast-seo/breadcrumbs",
+  "title": "Yoast Breadcrumbs",
+  "description": "Adds the Yoast SEO breadcrumbs to your template or content.",
+  "category": "yoast-internal-linking-blocks",
+  "icon": "admin-links",
+  "keywords": [
+    "SEO",
+    "breadcrumbs",
+    "internal linking",
+    "site structure"
+  ],
+  "textdomain": "wordpress-seo",
+  "attributes": {},
```

Comment:
You override the attributes in PHP, but not in JS. Shouldn't the className be here instead?

Reply:
you have a point, I got confused.